runtime/doc/eval.txt

```diff
@@ -1552,6 +1552,17 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 					operation.
 			regtype		Type of register as returned by
 					|getregtype()|.
+			completed_item    Current selected complete item on
+					|MenuPopupChanged|, Is `{}` when no complete
+					item selected.
+			height 		Height of popup menu on |MenuPopupChanged|
+			width   	width of popup menu on |MenuPopupChanged|
+			row  	 	Row count of popup menu on |MenuPopupChanged|,
```

Comment:
do we also want to add total number of items called `count`?

Reply:
If it's easy to get that number then yes.

Reply:
Added `size` to `v:event`